docs/openhabian.md

```diff
@@ -139,11 +139,19 @@ For the setup on Wi-Fi, you'll need to make your SSID and password known to the
 Additionally to the setup instructions given above, the following steps are needed:
 
 - Flash the system image to your micro SD card as described, do not remove the SD card yet
-- Access the first SD card partition from the file explorer of your choice (e.g. Windows file explorer)
+- Access the first SD card partition using the file explorer. It's a vfat (Windows) filesystem.
+  (we assume you're using a Windows, Mac or other desktop system to flash the SD)
 - Open the file `openhabian.conf` in a text editor
-- Uncomment and fill in `wifi_ssid="My Wi-Fi SSID"` and `wifi_psk="password123"`
+- Uncomment and complete the lines reading `wifi_ssid="My Wi-Fi SSID"` and `wifi_psk="password123"`
 - Save, Unmount, Insert, Boot
-- Continue with the instructions for the Raspberry Pi or Pine A64
+- Continue with the instructions for your hardware
+
+{: #ipv6-notes}
+### IPv6 notes
+
+You might encounter problems when you make use of IPv6 of some sort. openHABian installation may stop or hang forever.
```

Comment:
"You might encounter problems when you make use of IPv6 of some sort." could probably be better said: "You might encounter problems when you make use of IPv6 on some networks and systems."

Reply:
done